configure.ac

```diff
@@ -110,6 +115,9 @@ fi
 if [[ "x$HAVE_ANDROID" = "xyes" ]]; then
   FRIDA_VALAFLAGS="$FRIDA_VALAFLAGS -D ANDROID"
 fi
+if [[ "x$HAVE_QNX" = "xyes" ]]; then
+  FRIDA_VALAFLAGS="$FRIDA_VALAFLAGS -D QNX"
+fi
```

Comment:
Would be good to move this after DARWIN just below it (so the order is consistent with earlier).